src/apis/Cart.ts

```diff
@@ -0,0 +1,53 @@
+import {BucketType} from '@/types/Bucket';
+const dummyCartList: BucketType[] = [
+  {
+    market: {
+      id: 1,
+      name: '가게이름이너무나길면두줄이찍히는데설마',
+      images: ['https://legacy.reactjs.org/logo-og.png'],
+    },
+    products: [
+      {
+        id: 1,
+        name: '김치',
+        image: 'https://legacy.reactjs.org/logo-og.png',
+        originalPrice: 10000,
+        discountPrice: 7000,
+        count: 3,
+        tags: ['예시'],
+      },
+      {
+        id: 2,
+        name: '깻잎',
+        image: 'https://legacy.reactjs.org/logo-og.png',
+        originalPrice: 5000,
+        discountPrice: 3000,
+        count: 3,
+        tags: ['예시'],
+      },
+      {
+        id: 3,
+        name: '간장게장',
+        image: 'https://legacy.reactjs.org/logo-og.png',
+        originalPrice: 20000,
+        discountPrice: 17000,
+        count: 3,
+        tags: ['예시'],
+      },
+    ],
+  },
+];
+
+export const getCartHistory = async (): Promise<BucketType[] | null> => {
+  try {
+    return new Promise(resolve => {
```

Comment:
장바구니에는 항상 하나의 가게의 데이터만 들어 있다는 것을 보장해야 되니 장바구니의 리턴값이 [] 가 아니라 BucketType 이어야 하지 않을까요.

```suggestion
export const getCartHistory = async (): Promise<BucketType | null> => {
  try {
    return new Promise(resolve => {
```